docs/features/dynamic-analysis.work.md

```diff
@@ -0,0 +1,12 @@
+Dynamic Analysis
+===============
+
+This feature enables instrumentation of binaries to collect information for dynamic analysis.
+
+--------------------
+
+TODO:
+
+- Generate code to compute MVIDs less often than once per invoked method.
+- Synthesize helper types to contain fields for analysis payloads so that methods of generic types can be instrumented.
+- Accurately identify which methods are tests.
```

Comment:
Should probably add that we need to check handling of constructor initializers and field initializers.